src/Product/ProductHelper.php

```diff
@@ -429,8 +429,7 @@ public function maybe_swap_for_parent_id( int $product_id ): int {
 	 *
 	 * @return WC_Product The parent product object or the given product object of it doesn't have a parent.
```

Comment:
Just a small typo left here:
```suggestion
	 * @return WC_Product The parent product object or the given product object if it doesn't have a parent.
```